tidb-cloud/tidb-cloud-sso-authentication.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Single sign-on Authentication
+summary: Learn how to sign in via your Google workspace account and your Github Accounts in TiDB Cloud Web console.
+---
+# SSO(single sign-on) Authentication
+
+This document describes how individual or corporate users can quickly log in to TiDB CLoud Web Console via Googol workspace and GitHub accounts. Since your identity is centrally hosted on the third-party Google and GitHub platforms, TiDB cloud does not support you to modify your account password and enable MFA after logging in TiDB Cloud via SSO authentication.If you want to know about logging in through user and password , please see [Password Authentication](/tidb-cloud/tidb-cloud-password-authentication.md)
+
+## Sign in with Google SSO 
+
+To Sign in with your Google Workspace, perform these steps:
+
+1. Go to the TiDB Cloud login page: [www.tidbcloud.com](https://tidbcloud.com/).
+
+2. In TiDB Cloud Console, click Sign in With Google.
+
+3. Jump to Google Workspace Console, enter your Google Username and Password to login.
+
+4. Login is successful, jump to Choose to agree to "Privacy Policy, Terms of Service"
```

Comment:
On the login page https://tidbcloud.com/, there is no "Privacy Policy, Terms of Service".
![image](https://user-images.githubusercontent.com/79440533/210912269-d298761f-f154-4631-87fa-60a4c7852ebe.png)

Reply:
after click sign in wtih google , and then complete the third partner authentication , we will enter a page as following picture
<img width="715" alt="Screen Shot 2023-01-06 at 1 53 44 PM" src="https://user-images.githubusercontent.com/22299783/210939145-112ad7b0-f21c-4ffa-b6fc-b854212216ad.png">

Reply:
this page need user to accept TiDB Cloud Terms for the first time to login TiDB Cloud via SSO

Reply:
OK. If so, let's make the info a note as below because it is only for first-time login users via Google SSO.